genienlp/predict.py

```diff
@@ -36,7 +36,7 @@
 import torch
 
 from . import models
-from .data.embeddings import load_embeddings
+from .data_utils.embeddings import load_embeddings
```

Comment:
Why so many file renames?

Reply:
it's more intuitive (e.g. data folder does not contain the data itself. Also easier to infer dependencies between files in the same package.